app/controllers/posts_controller.rb

```diff
@@ -6,6 +6,36 @@ def index
     @posts -= [@post]
   end
 
+  def new
+   posts
```

Comment:
Rather than calling `posts` in each method, use a `before_action` filter instead.